src/main/resources/templates/fragments/admin-registry-user-permissions.html

```
@@ -0,0 +1,50 @@
+<form
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)